src/getDataset.js

```diff
@@ -90,9 +91,12 @@ const requestMainDataset = async (res, fetchUrls) => {
  * @param {Response} res
  */
 const getDataset = async (req, res) => {
-  const query = queryString.parse(req.url.split('?')[1]);
-  if (!query.prefix) {
-    return helpers.handleError(res, `getDataset request must define a prefix`);
+  const query = req.query;
+
+  try {
+    assert(query.prefix);
+  } catch {
+    return res.status(400).send('getDataset request must define a prefix');
```

Comment:
Why `try/catch` with `assert()` instead of just keeping the `if`?

Reply:
I think this is a better question for #228 where this commit was introduced; the fact that it showed up in this PR is just an artifact of my not having rebased this branch onto `error-handling` after making changes over there. 

But to answer your question here -- no strong reason except that I like the forcefulness of "assert" in defensive programming, and I was trying to corral the sending of non-200 status codes into `catch` blocks as much as possible in the `getDataset` (etc) files. I'm happy to change it back to `if` if you'd prefer that.